.github/workflows/upload-content-files.yml

```diff
@@ -40,12 +45,39 @@ jobs:
           git commit -m "Updated form data from Contentful"
           git push origin main
 
-  push-forms-to-staging:
-    name: 🛳 Push forms to S3 (Staging)
+      - name: Commit answer filter changes
```

Comment:
Combine answer filters and form changes into the same commit as they are dependant on eachother.

Reply:
👍🏾 Going to check this again with Elsa.

Reply:
Just saw Jaye's reply. Yep, will do.